apps/beeswax/src/beeswax/conf.py

```diff
@@ -32,6 +32,59 @@
 
 LOG = logging.getLogger(__name__)
 
+HIVE_DISCOVERY_LLAP = Config(
```

Comment:
We could also add to https://github.com/cloudera/hue/blob/master/desktop/conf/pseudo-distributed.ini.tmpl#L1143 and https://github.com/cloudera/hue/blob/master/desktop/conf.dist/hue.ini#L1114